history/titles/k_ramkahen.txt

```diff
@@ -1,86 +1,454 @@
 k_ramkahen = {
-	800.1.1={
+	1.1.1={
+		succession_laws = { 
+			male_preference_law 
+			single_heir_succession_law
+		}
+		government = feudal_government
+	}
+	#Tahret dynasty
+	182.05.18={
+		holder=270213 #Harus[121003]
+	} 
+	205.08.18={
+		holder=270216 #Tebhotep[121003]
 		liege=e_uldum
 	}
+	229.11.11={holder=270221} #Chinrasoth[121003]
+	235.07.05={holder=270227} #Enjtehoph[121003]
+	261.03.18={holder=270230} #Jinjtassot[121003]
+	310.01.12={holder=270232} #Munbenhi[121003]
+	348.03.29={holder=270238} #Ninjter[121003]	
+	350.01.02={
+		holder=0
+	}
 }
 d_ramkahen = {
-	800.1.1={
-		liege=k_ramkahen
+	1.1.1={
+		succession_laws = { 
+			male_preference_law 
+			single_heir_succession_law
+		}
+		government = feudal_government
 	}
-}
-c_ramkahen = {
-	750.1.1={government=feudal_government}
-	800.1.1={
-		liege=d_ramkahen
+	#Tahret dynasty
+	25.1.1={holder=270200} #Gilhonha[121003]
+	63.10.13={holder=270201} #Ghabhorush[121003]
+	102.12.10={holder=270205} #Phonkhibax[121003]
+	158.01.09={holder=270209} #Bankhiram[121003]
+	182.05.18={
+		holder=270213 #Harus[121003]
+		liege=k_ramkahen
+	} 
+	205.08.18={holder=270216} #Tebhotep[121003]
+	229.11.11={holder=270221} #Chinrasoth[121003]
+	235.07.05={holder=270227} #Enjtehoph[121003]
+	261.03.18={holder=270230} #Jinjtassot[121003]
+	310.01.12={holder=270232} #Munbenhi[121003]
+	348.03.29={holder=270238} #Ninjter[121003]
+	#ramkahen dynasty
+	348.03.29={
+		holder=270043 #Chathushaam[121001]
+		liege = 0
+	} 
+	392.03.08={holder=270045} #Cholhozor[121001]
+	424.06.27={holder=270047} #Zorwusud[121001]
+	435.08.10={holder=270049} #Oenbatuum[121001]
+	463.10.09={holder=270057} #Taemkhokir[121001]
+	481.09.05={holder=270058} #Debosuuf[121001]
+	508.1.1 = {
+		change_development_level = 7
 	}
-}
-c_virdam = {
-	750.1.1={government=feudal_government}
-	800.1.1={
-		liege=d_ramkahen
+	525.11.05={holder=270062} #Chibebhef[121001]
+	542.10.29={holder=270067} #Mariqe[121001]
+	561.10.28={holder=270070} #Ithihor[121001]
+	574.03.08={holder=270073} #Phaoris[121001]
+	583.1.1 = {
+		change_development_level = 8
 	}
+	603.1.1 = {
+		change_development_level = 9
+	}	
+	609.10.16={holder=270075} #Muthesi[121001]
+	648.01.19={holder=270077} #Cranhisse[121001]
 }
+# Empty provinces
+#c_ramkahen = {
```

Comment:
It's not an empty province. It's right there.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
1559 and 1569 have the same names, the latter is the empty province I sought, that explains it